src/main/resources/policies/document-rechten.rego

```diff
@@ -45,66 +46,107 @@ onvergrendeld_of_vergrendeld_door_user {
 
 default lezen := false
 lezen {
-    { behandelaar, coordinator, recordmanager }[_].rol in user.rollen
+    { behandelaar, coordinator, recordmanager, beheerder }[_].rol in user.rollen
```

Comment:
I thought rights should be stacked by adding roles to users, therefore only a `behandelaar` should have the read rights, or am I wrong?